plugins/shared/info-panel/style.less

```diff
@@ -183,6 +183,19 @@
             padding: 10px 0 0;
             user-select: text;
 
+            &-code {
+                margin-top: @panelPadding;
+
+                code {
+                    display: -webkit-box;
+                    margin-top: 10px; 
+                    overflow: hidden;
+                    padding: 5px @panelPadding;
+                    word-wrap: break-word;
+                    -webkit-line-clamp: 15;
```

Comment:
I'm reluctant to bring these rules in as they'll break in non-webkit. How about we show the string as-is and just trim the string in JS land? If we limit the string to X characters then we won't need to bother ourselves with a max-height for this container.